app/modules/users/parameters.py

```diff
@@ -208,4 +207,43 @@ def replace(cls, obj, field, value, state):
                 #     # Access granted
                 #     pass
 
+        if field == User.profile_asset_guid.key:
+            value = cls.add_replace_profile_asset(value)
+
         return super(PatchUserDetailsParameters, cls).replace(obj, field, value, state)
+
+    @classmethod
+    def remove(cls, obj, field, value, state):
+        if field == User.profile_asset_guid.key:
+            obj.profile_asset_guid = None
+        return True
+
+    @classmethod
+    def add(cls, obj, field, value, state):
```

Comment:
did we agree that _both_ `add` and `replace` (ops) should both be supported in a case like this?